webpack.config.js

```diff
@@ -1,21 +1,27 @@
-const path = require("path");
-const webpack = require("webpack");
-const CopyPlugin = require("copy-webpack-plugin");
-const CompressionPlugin = require("compression-webpack-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
+import path from "path";
+import * as fs from "fs";
+import webpack from "webpack";
+import CopyPlugin from "copy-webpack-plugin";
+import CompressionPlugin from "compression-webpack-plugin";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import MiniCssExtractPlugin from "mini-css-extract-plugin";
+import CssMinimizerPlugin from "css-minimizer-webpack-plugin";
+import { fileURLToPath } from "url";
+import { config } from "dotenv";
 
-require("dotenv").config();
+const __dirname = fileURLToPath(new URL(".", import.meta.url));
+
+config();
 
 /** Read the II canister ID from dfx's local state */
-function readCanisterId() {
+async function readCanisterId() {
```

Comment:
not sure `async` was actually ever needed, is it?

Reply:
Good catch, thanks